### ISSUE TITLE
false positive when awaiting Bluebird promises

### ISSUE THREAD

**Issue by abenhamdine** [SonarSource/SonarTS#658]
It's very annoying since it's make SonarTS unusable if we use non-native Promise libraries.  

The error itself is weird because it seems to be a Tslint rule rather than a true SonarTS rule.
Currently, this rule is disabled in my tslint.json, and before installing SonarTS, the error was not surfacing.

Related to https://github.com/SonarSource/SonarTS/issues/468

**I want to report a bug.**

**SonarTS version:** 1.7<br>
**Node.js version:** 9.11<br>
**TypeScript version:** 2.9.1<br>

**SonarLint version and used IDE: VS Code SonarLint 1.3.0<br>
**TSLint version:** 5.9.1<br>

**Rule key:** S4123

**Reproducer** 
```
import * as bluebird from 'bluebird'
        (async () => {
            const foo = await bluebird.promisify((err, res) => { })

        })()
```

**Logs**
```
// logs of analysis if relevant
```

**Expected behavior**

No error

**Comment by abenhamdine** [comment]
related to https://github.com/SonarSource/SonarTS/issues/659
